fileseeker.go

```diff
@@ -191,15 +197,20 @@ func main() {
 func statikBFS() {
 	log.Debug("Loading teachings...")
 
-	teachingData, err := loadTeachings(*configFile)
+	var config, err = loadConfig()
+	if err != nil {
+		os.Exit(1)
+	}
+
+	teachingData, err := loadTeachings(config.TeachingsPath)
```

Comment:
Sarebbe meglio fare il parsing della config (e quindi chiamare loadConfig) solo una volta nel main. Se ti serve poi in altre funzione gliela puoi passare come parametro